conda/recipe/meta.yaml

```diff
@@ -93,7 +93,6 @@ requirements:
     - lmdb >=0.9.28
     - libtiff >=4.3.0
     - libsndfile >=1.0.31
-    - libtar >=1.2.20
```

Comment:
libtar provides a static variant and we link with it so we don't need that in runtime.